src/inverted_index.c

```diff
@@ -393,24 +394,26 @@ size_t InvertedIndex_WriteEntryGeneric(InvertedIndex *idx, IndexEncoder encoder,
   // this can happen with duplicate tags for example
   if (idx->lastId && idx->lastId == docId) return 0;
 
+  t_docId delta = 0;
```

Comment:
**uint32_t** delta = 0 ? 

I know its the same but above you chose to use uint32_t for delta and I think its better to keep it and not mixing 
What do you think?